src/canvas/font.cpp

```diff
@@ -331,19 +331,24 @@ uint16_t NanoFont::unicode16FromUtf8(uint8_t ch)
 {
 #ifdef CONFIG_SSD1306_UNICODE_ENABLE
     static uint16_t unicode = 0;
```

Comment:
Now it is better to move `unicode` variable inside `if (ch & 0x80)` branch. Please, check clang compiler results.